frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/create-local-volume-set.tsx

```diff
@@ -0,0 +1,274 @@
+import * as React from 'react';
+import { match as RouterMatch } from 'react-router';
+import {
+  ActionGroup,
+  Button,
+  Form,
+  FormGroup,
+  TextInput,
+  Radio,
+  Expandable,
+  TextInputTypes,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import {
+  resourcePathFromModel,
+  BreadCrumbs,
+  Dropdown,
+  resourceObjPath,
+  withHandlePromise,
+  HandlePromiseProps,
+  ButtonBar,
+} from '@console/internal/components/utils';
+import { history } from '@console/internal/components/utils/router';
+import { ListPage } from '@console/internal/components/factory';
+import { k8sCreate, referenceFor } from '@console/internal/module/k8s';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager';
+import { LocalVolumeSetModel, LocalVolumeModel } from '../../models';
+import { NodesSelectionList } from './nodes-selection-list';
+import { RowUIDMap, LocalVolumeSetKind } from './types';
+import { getSelectedNodeUIDs } from './utils';
+import './create-local-volume-set.scss';
+
+const volumeTypeOptions = Object.freeze({
+  'SSD/NVMe': 'SSD/NVMe',
+  HDD: 'HDD',
+});
+const volumeModeOptions = Object.freeze({
+  Block: 'Block',
+  Filesystem: 'Filesystem',
+});
+const volumeSizeUnitOptions = Object.freeze({
+  TiB: 'TiB',
+  GiB: 'GiB',
+});
```

Comment:
I would prefer enums.

Reply:
I need to keep the plain objects for future fields